modules/light-clients/09-localhost/types/client_state.go

```
@@ -42,6 +42,16 @@ func (cs ClientState) GetLatestHeight() exported.Height {
 	return cs.Height
 }
 
+// GetTimestampAtHeight returns 0. Localhost client has no consensus state.
+func (cs ClientState) GetTimestampAtHeight(
+	_ sdk.Context,
+	_ sdk.KVStore,
+	_ codec.BinaryCodec,
+	_ exported.Height,
+) (uint64, error) {
+	return 0, nil
```

This should probably return an error for now. The localhost doesn't work atm